bake.hcl

```diff
@@ -15,6 +15,10 @@ variable "FDB_LIB_URL" {
   default = "https://github.com/apple/foundationdb/releases/download/6.2.30/foundationdb-clients_6.2.30-1_amd64.deb"
 }
 
```

Comment:
Can we get rid of this bake file? The defaults for the dockerfile arguments are already in the dockerfile itself.